exporters/cinder.go

```diff
@@ -21,36 +24,85 @@ func NewCinderUsageExporter(db *sql.DB) (*CinderUsageExporter, error) {
 			"Total number of volumes per OpenStack project",
 			[]string{"project_id"}, nil,
 		),
-		size: prometheus.NewDesc(
+		volumesSize: prometheus.NewDesc(
 			"openstack_project_volume_size_gb",
 			"Total volume size in GB per OpenStack project",
 			[]string{"project_id"}, nil,
 		),
+		snapshots: prometheus.NewDesc(
+			"openstack_project_snapshots",
+			"Total number of snapshots per OpenStack project",
+			[]string{"project_id"}, nil,
+		),
+		backupsSize: prometheus.NewDesc(
+			"openstack_project_backups_size_gb",
+			"Total size of backups in GB per OpenStack project",
+			[]string{"project_id"}, nil,
+		),
+		totalBackups: prometheus.NewDesc(
+			"openstack_project_backups",
+			"Total number of backups per OpenStack project",
+			[]string{"project_id"}, nil,
+		),
 	}, nil
 }
 
 func (e *CinderUsageExporter) Describe(ch chan<- *prometheus.Desc) {
 	ch <- e.volumes
-	ch <- e.size
+	ch <- e.volumesSize
+	ch <- e.snapshots
+	ch <- e.backupsSize
+	ch <- e.totalBackups
 }
 
 func (e *CinderUsageExporter) Collect(ch chan<- prometheus.Metric) {
 	e.collectMetrics(ch)
 }
 
 func (e *CinderUsageExporter) collectMetrics(ch chan<- prometheus.Metric) {
-	rows, err := e.db.Query("SELECT project_id, COUNT(id) as total_volumes, SUM(size) as total_size_gb FROM volumes WHERE deleted = 0 GROUP BY project_id")
+	rows, err := e.db.Query(`
+		SELECT 
```

Comment:
perhaps we can just run multiple queries? Might be easier / quicker than to do so much joining